behat/tests/features-v2/reporting/sections/additional-information/additional-information.feature

```diff
@@ -0,0 +1,21 @@
+@v2
+Feature: Additional Information
+
+  Scenario: A user has no additional information to add
+    Given a Lay Deputy has not started a report
+    When I view and start the additional information report section
+    And there is no additional information to add
+    Then I should be on the additional information summary page
+    And the additional information summary page should contain the details I entered
+
+  Scenario: The section navigation links are correctly displayed
+    Given a Lay Deputy has not started a report
+    When I view the additional information report section
+    Then the previous section should be "Actions"
+    And the next section should be "Documents"
```

Comment:
I'm starting to wonder if maybe we should have a single folder for checking the navigation links for each section so they're kept together...although I'm not sure if that would make things less or more confusing. What do you think?